examples/hello_worlds.rb

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env ruby
+
+require 'vedeu'
```

Comment:
As long as I do not have the development dependencies installed I can neither get it running with `bundle exec` nor `ruby example/hello_worlds.rb`, because `bundler/setup` is required somewhere within the vedeu/all, I suppose. If I uncomment, I get

    examples/hello_worlds.rb:5:in `<main>': 0.6.21 (RuntimeError)